### DISCUSSION TITLE
Add all comment's messages to my release

### DISCUSSION THREAD

**Discussion by evertramos** [release-drafter/release-drafter#1043] [Q&A] [answered]
Hi there!

I could not find how to add all my commit messages to my release, is there a way we can do that? I tried the example (basic and advanced) as in the docs, but did not load the commit messages.

Here is what I have used:

[My release drafter - Basescript](https://github.com/evertramos/basescript/blob/cd52b5127327c42a054a9d45f5d3ca05e8a4c508/.github/release-drafter.yml)

Thanks!

**Comment by jetersen** [comment, marked as answer]
We only support release notes based on pull requests as they are able to have labels that support the category sorting.

**Reply by jetersen** [reply]
Again there are other benefits to putting most things in a PR. You get CI run on that PR. You can set up branch protection and enforce your CI passes and enable auto merge once CI passes to speed up workflows 😄

Also there are benefits of putting things into commits. When they are so small that they do not need to go into release notes.

**Reply by evertramos** [reply]
Thanks for you attention @jetersen I understand your point and agree! I use bash script quiet a lot and have a tons of script that already does some good tricks around that, but will update those for github actions and use your draft-release for that, just struggling a little bit in this beggining. Thanks once again!
